src/caller.jl

```diff
@@ -49,6 +49,32 @@ function convert_ring_content(value_list, rng)
     return return_dict
 end
 
+function create_ring_from_singular_ring(r::libSingular.ring_ptr)
+   ordering = libSingular.ring_ordering_as_symbol(r)
+   c = libSingular.rCoeffPtr(r)
+   if libSingular.nCoeff_is_Q(c)
```

Comment:
I suggest we also wrap `getCoeffType`, which is also useful for interactive debugging